usecases/auth/authorization/controller.go

```diff
@@ -16,11 +16,12 @@ import (
 )
 
 type Controller interface {
-	CreateRoles(roles ...*models.Role) error
+	UpsertRoles(roles ...*models.Role) error
```

Comment:
**thought:** i think we shall test the behaviour of casbin here, because the original thought was to add to exiting policy